source/gcp/prerequisites/considerations.rst

```diff
@@ -83,3 +83,7 @@ Below there is an example of a configuration that uses more than one bucket:
     </bucket>
 
   </gcp-bucket>
+  
+.. note::
+   Once the configuration is complete, you can enable the `Security Information` visualization feature in the **Wazuh Dashboard**. You can read more about it `here <https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/settings.html#modules>`_.
+  
```

Comment:
Like in AWS, the note here has a lack of visibility. I suggest adding a new section with this content.